uint8 gACT 
uint8 gGTO 
uint8 gSTA 
uint8 gOBJ 
uint8 gFLT
uint8 gPR
uint8 gPO
uint8 gCU
